Realm/RLMSchema.mm

```diff
@@ -71,66 +92,92 @@ + (void)initialize {
     }
     initialized = true;
 
-    NSMutableArray *schemaArray = [NSMutableArray array];
-    RLMSchema *schema = [[RLMSchema alloc] init];
+    s_localNameToClass = [NSMutableDictionary dictionary];
```

Comment:
These can be initialized where they're declared and the `initialize` method can be removed entirely.